crates/forge-runner/src/running.rs

```diff
@@ -187,6 +188,7 @@ fn build_syscall_handler<'a>(
 pub struct RunResultWithInfo {
     pub(crate) run_result: Result<RunResult, RunnerError>,
     pub(crate) gas_used: u128,
+    pub(crate) used_resources: UsedResources,
```

Comment:
I would advise to merge those 2 into just one `resources`

Reply:
Can't really put `gas_used` inside of `UsedResources`, as gas is calculated later, and it uses `UsedResources`. There could be a new struct `Resources` these 2, but I don't know if it wouldn't just introduce complexity.

Reply:
If it's not feasible i'm not going to push it